m2cgen/assemblers/boosting.py

```diff
@@ -10,6 +10,7 @@
 class BaseBoostingAssembler(ModelAssembler):
 
     classifier_names = {}
+    strided_layout_for_multiclass = True
```

Comment:
Forgive me my ignorance but what exactly `strided` mean in this context? :)

Reply:
You shouldn't apologize! I was really struggling with the good name for this parameter! 😄 
 Initially I thought about `step` word in the name. Then `stride` came to my mind from convolutional nets. I started to google for any synonyms and came to the following picture

![image](https://user-images.githubusercontent.com/25141164/82379805-35cdaa00-9a30-11ea-92cc-3869f0feee79.png)
(taken from 3rd slide of https://slideplayer.com/slide/9359236/)

Reply:
Got it, thank you!